src/CddInterface.c

```diff
@@ -444,23 +331,14 @@ static dd_MatrixPtr GapInputToMatrixPtr(Obj input)
     ErrorMayQuit("This should not happen, please report this!", 0, 0);
   }
 
-  str_len = GET_LEN_STRING(string);
+  int str_len = GET_LEN_STRING(string);
   //fprintf(stdout, "%d: ", str_len);
   //ErrorMayQuit( "j", 0, 0 );
 
   char k_matrix[str_len];
-  strcpy(k_linearity_array, PLIST_STR(ELM_PLIST(input, 6)));
-
-  if (k_numtype == 3)
-  {
-    strcpy(k_matrix, PLIST_STR(ELM_PLIST(input, 7)));
-  }
-  else
-  {
-    strcpy(k_matrix, RATPLIST_STR(ELM_PLIST(input, 7)));
-  }
-
-  strcpy(k_rowvec, RATPLIST_STR(ELM_PLIST(input, 9)));
+  strcpy(k_linearity_array, CONST_CSTR_STRING(ELM_PLIST(input, 6)));
```

Comment:
Using **CONST_CSTR_STRING** raise warning error when compiling and tests can not be completed:
symbol lookup error: /home/kamal/Documents/local_gap_packages/pkg/CddInterface/bin/x86_64-pc-linux-gnu-default64-kv3/CddInterface.so: undefined symbol: CONST_CSTR_STRING.
I am using GAP 4.10.2 of 19-Jun-2019.
But changing CONST_CSTR_STRING to CSTR_STRING solves the problem.

Reply:
Ah, right, I forgot that `CONST_CSTR_STRING` is new in master / GAP 4.11. I'll fix the PR ASAP.